src/NUnitConsole/nunit3-console/ConsoleOptions.cs

```diff
@@ -33,9 +32,19 @@ namespace NUnit.Common
     /// </summary>
     public class ConsoleOptions : CommandLineOptions
     {
+        private readonly IFileSystem fileSystem;
```

Comment:
Our standard naming is to use `_fileSystem` for this and not to use `this` in references.